Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabLoader.h

```diff
@@ -106,6 +108,16 @@ namespace AzToolsFramework
             static bool IsValidPrefabPath(AZ::IO::PathView path);
 
         private:
+            /**
+             * Generate a template dom in the proper format to be saved to disk.
+             * @param templateRef The template whose dom we want to generate.
```

Comment:
nit: remnant of old incorrect comments I think: We are not generating this dom. We are using this dom to generate a new dom that uses the file format we need

Reply:
Hum I think "generate" in this context isn't completely wrong, but I adjusted the comment to be more precise.